src/wallet/test/fuzz/crypter.cpp

```diff
@@ -27,36 +27,36 @@ FUZZ_TARGET(crypter, .init = initialize_crypter)
     // These values are regularly updated within `CallOneOf`
     std::vector<unsigned char> cipher_text_ed;
     CKeyingMaterial plain_text_ed;
-    const std::vector<unsigned char> random_key = ConsumeRandomLengthByteVector(fuzzed_data_provider);
+    const std::vector<unsigned char> random_key = ConsumeFixedLengthByteVector(fuzzed_data_provider, WALLET_CRYPTO_KEY_SIZE);
 
     LIMITED_WHILE(good_data && fuzzed_data_provider.ConsumeBool(), 10000)
     {
         CallOneOf(
             fuzzed_data_provider,
             [&] {
-                const std::string random_string = fuzzed_data_provider.ConsumeRandomLengthString();
+                const std::string random_string = fuzzed_data_provider.ConsumeRandomLengthString(100);
                 SecureString secure_string(random_string.begin(), random_string.end());
 
                 const unsigned int derivation_method = fuzzed_data_provider.ConsumeBool() ? 0 : fuzzed_data_provider.ConsumeIntegral<unsigned int>();
 
                 // Limiting the value of nRounds since it is otherwise uselessly expensive and causes a timeout when fuzzing.
                 crypt.SetKeyFromPassphrase(/*strKeyData=*/secure_string,
-                                           /*chSalt=*/ConsumeRandomLengthByteVector(fuzzed_data_provider),
+                                           /*chSalt=*/ConsumeFixedLengthByteVector(fuzzed_data_provider, WALLET_CRYPTO_SALT_SIZE),
                                            /*nRounds=*/fuzzed_data_provider.ConsumeIntegralInRange<unsigned int>(0, 25000),
                                            /*nDerivationMethod=*/derivation_method);
             },
             [&] {
-                const std::vector<unsigned char> random_vector = ConsumeFixedLengthByteVector(fuzzed_data_provider, 32);
+                const std::vector<unsigned char> random_vector = ConsumeFixedLengthByteVector(fuzzed_data_provider, WALLET_CRYPTO_KEY_SIZE);
                 const CKeyingMaterial new_key(random_vector.begin(), random_vector.end());
                 const std::vector<unsigned char>& new_IV = ConsumeFixedLengthByteVector(fuzzed_data_provider, 16);
```

Comment:
If we're using constants from `crypter.h`, then this can be changed to `WALLET_CRYPTO_IV_SIZE` for consistency.

Reply:
Sure, I'll address it.